README.md

```diff
@@ -8,92 +8,193 @@
 Metaprogramming tools for DataFrames.jl objects.
 These macros improve performance and provide more convenient syntax.
 
-# Features
+DataFrames.jl has the functions `select`, `transform`, and `combine` 
+for manipulating data frames. DataFramesMeta provides the macros 
+`@select`, `@transform`, and `@combine` to mirror these functions with 
+more convenient syntax. Inspired by [dplyr](https://dplyr.tidyverse.org/) in R 
+and [LINQ](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/)
+in C#. 
+
+In addition, DataFramesMeta provides 
+
+* `@orderby`, for sorting data frames
+* `@where`, for keeping rows of a DataFrame matching a given condition
+* `@by`, for grouping and combining a data frame in a single step
+* `@with`, for working with the columns of a data frame with high performance and 
+  convenient syntax
+* `@eachrow`, for looping through rows in data frame, again with high performance and 
+  convenient syntax. 
+* `@linq`, for piping the above macros together, similar to [magrittr](https://cran.r-project.org/web/packages/magrittr/vignettes/magrittr.html)'s
+  `%>%` in R. 
+
+To reference columns inside DataFramesMeta macros, use `Symbol`s. For example, use `:x`
+to refer to the column `df.x`. To use a variable `varname` representing a `Symbol` to refer to 
+a column, use the syntax `cols(varname)`. 
+
+Use `passmissing` from [Missings.jl](https://github.com/JuliaData/Missings.jl) to 
+propagate `missing` values more easily. 
```

Comment:
```suggestion
Use `passmissing` from [Missings.jl](https://github.com/JuliaData/Missings.jl) to 
propagate `missing` values more easily. 
```

This feels really odd to read. I mean, yeah it's general advice not related to DataFramesMeta.jl directly. Propose to remove. Or say "Hint:"

Reply:
It's re-exported by DataFrames, which is in turn re-exported by DataFramesMeta. So they don't have to download Missings.jl. I've re-written it to be more relevant.

Reply:
The main motivation is that I *really* want people to know about `passmissing` because people routinely get burned and don't know how to fix it.

Reply:
For what it's worth, it is not obvious how to use it with `@linq`...